doc/source/cluster/cluster_under_construction/ray-clusters-on-vms/user-guides/community-supported-cluster-manager/index.rst

```diff
@@ -9,6 +9,13 @@ Community Supported Cluster Managers
 
     If you're using AWS, Azure or GCP you can use the :ref:`Ray Cluster Launcher <cluster-cloud>` to simplify the cluster setup process.
 
+
+To use Ray autoscaling on other Cloud providers or cluster managers, you can implement the ``NodeProvider`` interface (100 LOC) 
```

Comment:
I think this page should first serve users who want an index of available options, then secondarily provide information on creating a new node provider. So the TOC should go first, then the custom node provider info as a subsection after the TOC. Something like this:


```
> Community-supported Cluster Managers
    Note: If you’re using AWS, Azure or GCP you can use the [Ray Cluster Launcher](https://ray--27431.org.readthedocs.build/en/27431/cluster/cloud.html#cluster-cloud) to simplify the cluster setup process.
    The following is a list of community supported cluster managers:
    <TOC>
>> Implementing a custom node provider
    To use Ray autoscaling on other Cloud providers or cluster managers, you can implement the NodeProvider interface (100 LOC) and register it in [node_provider.py](https://github.com/ray-project/ray/tree/master/python/ray/autoscaler/node_provider.py). Contributions are welcome!
```